docs/2.8.0/docs/daml-finance/reference/patterns.rst

```diff
@@ -77,9 +77,10 @@ We can then fetch an interface "by key" by
 
 Similarly, this pattern also lets us exercise a choice on an interface "by key".
 
-The Reference pattern is currently used in Daml Finance for instruments and accounts, where we
-ensure that a template and its companion `Reference` template are kept in sync. It is however
-important to understand this pattern should you implement custom instruments or accounts.
+The Reference pattern is currently used in Daml Finance for instruments, accounts, and the holding
+factory, where we ensure that a template and its companion `Reference` template are kept in sync.
```

Comment:
```suggestion
factory, where a template and its companion `Reference` template are kept in sync.
```

Reply:
I changed the this paragraph to:

The Reference pattern is currently used in Daml Finance for instruments, accounts, and the holding
factory. To maintain synchronization between a template and its corresponding `Reference`, we 
simultaneously create or archive instances of both. Additionally, if the contract id of the target 
template changes, we update the `Reference` instance accordingly.

It is important to understand this pattern should you implement custom instruments, accounts or 
holding factories.